### PR TITLE
Update readme with new copy

### PR DESCRIPTION
**Pull request by markbiek**
Closes https://github.com/Automattic/dotcom-forge/issues/8731

# Testing Instructions

Verify the copy matches the final version [here](https://github.com/Automattic/dotcom-forge/issues/8731#issuecomment-2310692172).

We'll update the changelog once we have a final version of the plugin.

### CODE REVIEW

File: src/readme.txt [hunk 1]
```diff
@@ -1,23 +1,63 @@
...
+Requires at least: 6.3
+Requires PHP: 7.0
+Tested up to: 6.4
+Stable tag: 2.0.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
+A WordPress plugin that helps users to migrate their sites to WordPress.com
+
 == Description ==
```
Comment (fditrapani) [inline src/readme.txt:13]: We might want to make this a title (## Description) to match the other titles we have on the page.

File: src/readme.txt [hunk 2]
```diff
@@ -1,23 +1,63 @@
...
+- **Built-In security**
+  Benefit from round-the-clock security monitoring, DDoS protection, real-time backups, and hassle-free SSL provisioning.
+- **Automatic updates**
+  Stay up-to-date with the latest WordPress versions automatically and keep your site secure and performant.
+
+== Installation ==
 
-== FREQUENTLY ASKED QUESTIONS ==
+1. To begin, click on the Plugins link in the left hand sidebar, then click Add New.
+2. Search for Move to WordPress.com. The latest version will be in the search results. Click the Install Now button.
```
Comment (fditrapani) [inline src/readme.txt:50]: This should be "Migrate to WordPress.com"

File: src/readme.txt [hunk 3]
```diff
@@ -1,23 +1,63 @@
...
+3. Next, click the Activate button. After activating, you can navigate to Move to WordPress.com in your admin area.
```
Comment (fditrapani) [inline src/readme.txt:51]: "Migrate to WordPress.com"

File: src/readme.txt [hunk 4]
```diff
@@ -1,23 +1,63 @@
...
+- **No downtime**
+  Keep your site live during migration, ensuring uninterrupted service for your visitors.
+- **Developer-friendly tools**
+  Retain full control with access to WP-CLI, SSH, GitHub deployments, and staging environments.
+- **Built-In security**
+  Benefit from round-the-clock security monitoring, DDoS protection, real-time backups, and hassle-free SSL provisioning.
+- **Automatic updates**
+  Stay up-to-date with the latest WordPress versions automatically and keep your site secure and performant.
+
+== Installation ==
```
Comment (fditrapani) [inline src/readme.txt:47]: Can we make this a title (## Installation) instead of using this text formatting?

**Review by fditrapani — COMMENTED** [review]
Thanks for getting this started. I left some comments about the formatting otherwise. This looks great.

File: src/readme.txt (hunk 1 above)
Comment (markbiek) [reply src/readme.txt:13]: This matches the format of the existing readme.txt for the Move to WordPress plugin (https://mc.a8c.com/pb/3581f/#plain).

I think it has to be that way to meet the .org [plugin readme standards.](https://wordpress.org/plugins/readme.txt)

File: src/readme.txt (hunk 2 above)
Comment (markbiek) [reply src/readme.txt]: Fixed!

File: src/readme.txt (hunk 3 above)
Comment (markbiek) [reply src/readme.txt]: Fixed!

File: src/readme.txt [hunk 5]
```diff
@@ -1,23 +1,63 @@
...
 
-== FREQUENTLY ASKED QUESTIONS ==
+1. To begin, click on the Plugins link in the left hand sidebar, then click Add New.
+2. Search for Move to WordPress.com. The latest version will be in the search results. Click the Install Now button.
+3. Next, click the Activate button. After activating, you can navigate to Move to WordPress.com in your admin area.
 
-=Why do you need my email?=
-We require an email address to send you updates on the migration process, notify you of any errors that occur during the migration.
+== Changelog ==
```
[the remark this answers is not in the document]
Comment (markbiek) [reply src/readme.txt:53]: This is another place where I think it's supposed to be this way to match the plugin readme standards.

File: src/readme.txt (hunk 1 above)
Comment (fditrapani) [reply src/readme.txt:13]: Aah ok. Sorry to have you do this. We should likely update the other titles to match this formatting to be consistent then.

Comment (markbiek) [reply src/readme.txt:13]: Good point, I didn't think of that when I was looking at it 😝 . I'll fix those shortly.

Comment (markbiek) [reply src/readme.txt:13]: Fixed!

File: src/readme.txt [hunk 6]
```diff
@@ -1,23 +1,63 @@
-=== WordPress.com Migration ===
-Contributors: wordpress.com, blogvault, akshatc
-Tags: wordpress.com, migration
-Requires at least: 4.0
-Tested up to: 6.6
-Requires PHP: 5.6.0
-Stable tag: 5.65
+=== Move to WordPress.com ===
```
[the remark this answers is not in the document]
Comment (markbiek) [reply src/readme.txt]: I wasn't sure about that one. But I guess that falls under the research you did where change the name/title is fine as long as we don't touch the plugin slug (`wpcom-migration`).

Fixed!

**Review by fditrapani — APPROVED** [review]
Thanks for the quick updates! Looks great now.

**Pull request merged**